timemachine/cpp/src/kernels/k_neighborlist.cuh

```diff
@@ -234,7 +234,7 @@ void __global__ k_find_blocks_with_ixns(
 
     if (include_col_block) {
 
-        // Compute center of column box and extent coords.
+        // Compute center of column block and extent coords.
```

Comment:
Some imprecise comments @mcwitt pointed out while going over nonbonded, could be pulled out into separate PR